tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -392,21 +399,51 @@ def _execute_test_suite(self, suite: TestSuiteDescription) -> bool:
                         agent_test_logger.info("\t%s", r)
                     agent_test_logger.info("")
 
-                    if len(failed) == 0:
-                        message.status = TestStatus.PASSED
-                    else:
-                        message.status = TestStatus.FAILED
-                        message.message = f"Tests failed: {failed}"
-
                 except:  # pylint: disable=bare-except
```

Comment:
Do we also need to add `failed = True` in this except block? Although it might not be necessary because the except blocks above handle this

Reply:
ah, good catch, thanks! yes, it is needed (and important, since an exception at this level of the code may prevent some tests from running and we need to be notified about that). I'm adding it now.

Reply:
added